src/index.jsx

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import { render } from 'react-dom';
 import './style.css';
 
+import Header from './Components/Header';
+
+import Dashboard from './Components/Dashboard';
+
+import smartHomeData from './smartHomeData';
+
 const App = () => (
   <>
-
+    <div className="container">
+      <Header title="Chytrý dům" />
+      <Dashboard data={smartHomeData} />
```

Comment:
Jen tak btw, v Dashboard tenhle props nečteš. Tj buď ho budeš posílat takhle tady, a v Dashboard si ta data přečteš z toho props, a nebo to tady odmažeme a můžeš si ta data importovat v Dashboard  jak to tam děláš aktuálně 👍